templates/bashrc.osh-template

```diff
@@ -1,5 +1,5 @@
 # Path to your oh-my-bash installation.
-export OSH=$HOME/.oh-my-bash
+export OSH="$HOME/.oh-my-bash"
```

Comment:
Quotes not mandatory either in this context. The word splitting and pathname expansions don't occur either in argument of the form of the variable assignments for the special assignment builtins (such as `export`, `declare`, `local`, `typeset`, and `readonly`).